src/openforms/js/components/form/fieldset.js

```diff
@@ -28,20 +28,17 @@ const FIELDSET_BASIC = {
             label: 'Clear on hide',
             tooltip: 'Remove the value of this field from the submission if it is hidden',
         },
+        {
+            type: 'checkbox',
+            key: 'showHeader',
+            label: 'Show fieldset header',
+            tooltip: 'Show a line and the label above the fieldset in the form',
+            defaultValue: true,
```

Comment:
as discussed, please invert to `hideHeader` rather than `showHeader` to not mess up existing forms